src/components/Navigation.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
+          <Link className="link" to="/"> 
           <a href="">Home</a>
+          </Link>
         </div>
         <div>
+        <Link className="link" to="/about"> 
           <a href="">About</a>
+          </Link>
         </div>
```

Comment:
The `Link` component renders an anchor tag so you don't need to nest an `a` tag inside of it. 

```
<Link to"/">Home</Link>
```